src/components/VxpLink.native.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <Label v-if="externalUrl" class="vxp-link-button typ-body" :text="text" @tap="onTap" />
+  <Label v-else :text="text" class="vxp-link-button typ-body" @tap="$emit('tap', $event)" />
+</template>
+
+<script>
+import { openUrl } from 'tns-core-modules/utils/utils'; // eslint-disable-line
```

Comment:
please move tns imports to another file like utils.js, you can check nuweb-ui for sample implementation.

Reply:
These imports are moved under utils.js & utils.native.js